README.md

```diff
@@ -30,6 +30,17 @@ All nodes connected to an elastic ip will be contained in the public subnet (Bon
 `existing_public_subnet_id` is the manager's public subnet id.
 `existing_private_subnet_id` is the manager's private subnet id.
 
+## Azure
+This blueprint assumes an azure cloudify manager with rsa-key-blueprint installed on.
+For doing so, see the [cloudify-key-plugin](https://github.com/cloudify-examples/cloudify-key-plugin/tree/master/examples) example.
```

Comment:
Instead say something like
"""
If you do not already have a key pair on your Azure manager, you can create one using the [rsa-key-blueprint]( https://raw.githubusercontent.com/cloudify-examples/cloudify-key-plugin/master/examples/rsa-key-blueprint.yaml).

Use the private key path to populate your ```azure_agent_key_file``` input and your outputs to populate the key data in the ```vm_os_pubkeys``` input.
""""